code/index.html

```diff
@@ -1,34 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
-    <title>Chat Bot</title>
+      rel="stylesheet"
+    />
+    <title>your daily affirmation</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <div class="video-container">
+      <video autoplay loop muted>
+        <source
+          src="assets/pexels-apoorv-srivastava-5338469.mp4"
```

Comment:
I love the moving background and the way you have styled it under the chat area! How did you do that?

Reply:
Thank you! It was actually very easy! I just set the background image to linear-gradient(rgba(0, 0, 0, 0), rgba(218, 9, 204, 0.5)),
    and then the same for the main.

Reply:
Thanks for the help! I'll have to try it sometime!